checkouts-documentation.cfg

```diff
@@ -47,22 +43,23 @@ documentation               = git ${remotes-https:plone}/documentation.git egg=f
 buildout.coredev            = git ${remotes-https:plone}/buildout.coredev.git egg=false depth=1 branch=5.0
 ansible-playbook            = git ${remotes-https:plone}/ansible-playbook.git egg=false depth=1 branch=STABLE
 bobtemplates.plone          = git ${remotes-https:plone}/bobtemplates.plone.git egg=false depth=1 branch=master
-plone.api                   = git ${remotes-https:plone}/plone.api.git egg=false depth=1 branch=master
 collective.transmogrifier   = git ${remotes-https:collective}/collective.transmogrifier.git egg=false depth=1 branch=master
 
+# these are coming from plone releases, and their tags should be updated when newer Plone versions come out
+diazo                       = git ${remotes-https:plone}/diazo.git egg=false depth=1 branch=1.2.3
```

Comment:
1.2.3 is not a branch but a tag, so it should be `rev=1.2.3`.
And `depth=1` does not work, unless tag 1.2.3 matches the very latest commit on master.